src/js/game.js

```diff
@@ -0,0 +1,195 @@
+      return name.slice(0, 5);
+    }
+  }
+
+  function preventInput(type) {
+    if (type === INPUT_TYPE.NAME) {
+      namesInput.disabled = true;
+      nameSubmitBtn.disabled = true;
+      return;
+    }
+
+    if (type === INPUT_TYPE.GAMETIMES) {
+      const gameTimes = $("#times");
+      const gameTimesInput = $("#times input");
+      const gameTimesSubmitBtn = gameTimes.querySelector("button");
+      gameTimesInput.disabled = true;
+      gameTimesSubmitBtn.disabled = true;
+      return;
+    }
+  }
+
+  function showGameTimesInput() {
+    const gameTimesField = form.lastElementChild;
+    gameTimesField.innerHTML = `
+    <p>시도할 횟수를 입력해주세요.</p>
+    <div id="times" class="d-flex">
+      <input type="number" class="w-100 mr-2" placeholder="시도 횟수" />
+      <button type="button" class="btn btn-cyan">확인</button>
+    </div>
+    `;
+    form.appendChild(gameTimesField);
+  }
+
+  function onGameTimesSubmit() {
+    const gameTimes = $("#times");
+    const gameTimesInput = $("#times input");
+    const gameTimesSubmitBtn = gameTimes.querySelector("button");
+    gameTimesSubmitBtn.addEventListener("click", () => {
+      times = gameTimesInput.value;
+
+      if (times === "0") {
+        return alert(ERROR_MESSAGE.TIMES_ERROR);
+      }
+
+      preventInput(INPUT_TYPE.GAMETIMES);
+      showPlayCars();
+    });
+  }
+
+  function showPlayCars() {
+    const nameCard = document.createElement("div");
+    nameCard.setAttribute("class", "mt-4 d-flex");
+    const nameCardComponents = carNames
+      .map(
+        name => `
+        <div class="mr-2" data-name=${name}>
+          <div class="car-player">${name}</div>
+        </div>`
+      )
+      .join("");
+    nameCard.innerHTML = `
+    ${nameCardComponents}
+    `;
+    nameCards.appendChild(nameCard);
+    playGames(times);
+    pickWinner();
+    showWinner();
+  }
+
+  function playGames(times) {
+    for (let i = 0; i < times; i++) {
+      calculateDistance();
+    }
+  }
+
+  function calculateDistance() {
+    let km = 0;
+    for (let i = 0; i < carNames.length; i++) {
+      km = Math.floor(Math.random() * 10);
+      if (!carsDistance[i] && carsDistance !== 0) {
+        carsDistance[i] = 0;
+      }
+
+      if (km >= 4) {
+        carsDistance[i]++;
+        showMovedCars(carNames[i]);
+      }
+    }
+  }
+
+  function showMovedCars(name) {
+    const carCard = $(`.mr-2[data-name="${name}"]`);
+    const moveIcon = document.createElement("div");
+    moveIcon.setAttribute("class", "forward-icon mt-2");
+    moveIcon.textContent = "⬇️";
+    carCard.appendChild(moveIcon);
+  }
+
+  function pickWinner() {
+    const maxDistance = carsDistance.reduce((previous, current) => {
+      return previous > current ? previous : current;
+    });
+    let index = carsDistance.indexOf(maxDistance);
+    while (index !== -1) {
+      winner.push(carNames[index]);
+      index = carsDistance.indexOf(maxDistance, index + 1);
+    }
+  }
```

Comment:
1. car의 distance값과 name이 매칭이 되니까 처음에 객체로 묶었으면 계산하기 편했을 것 같습니다~
2. `Math.max()`를 최대값을 구할 때 쓰면 편합니다 :)
```suggestion
  function pickWinner() {
    const maxDistance = Math.max(...carsDistance);
    const winnerNames = carsDistance.reduce((names, cur, idx) => {
      if (cur !== maxDistance) return names;
      names.push(carNames[idx]);
      return names;
    }, []);
    winner.push(...winnerNames);
  }
```